std/http/http_status.ts

```diff
@@ -126,6 +126,13 @@ export enum Status {
   NotExtended = 510,
   /** RFC 6585, 6 */
   NetworkAuthenticationRequired = 511,
+
+  /** RFC 2616, 6.1.1 */
+  Informational = 1,
+  Success = 2,
+  Redirection = 3,
+  ClientError = 4,
+  ServerError = 5,
```

Comment:
Hmm I don't like the status code for a particular block of errors is being conflated with actual status codes. It looks like there's a status code 1 now. Maybe these can just be moved to a different enum.